lib/scrabble.rb

```diff
@@ -1,4 +1,14 @@
+require 'pry'
+
 class Scrabble
+
+  def score(word)
+    return 0 if word == "" || word == nil
+    return "Invalid Input" if not word.class == String
+    return point_values[word.upcase] if word.length == 1
+    score_with_no_multipliers(word)
```

Comment:
Like this approach. Nice that you were able to extract this functionality into supporting methods.